### DISCUSSION TITLE
Moving away from swell third party cookies for google chrome update

### DISCUSSION THREAD

**Discussion by vettloffah** [swellstores/community#512] [Help]
As [google announced](https://developers.google.com/privacy-sandbox/3pcd), in the 2nd half of 2024 they will automatically disable 3rd party cookies in Chrome. Currently, a site audit shows a couple 3rd party cookies being used in the swell lib in order for the `cart` and `account` endpoint requests.

Does anyone know how we make sure that our sites won't break when this change happens in a few months? 

Perhaps we can use the `.getSession` and `setSession` functions to manage the cookies on our own domain?

**Comment by vettloffah** [comment]
After closer inspection I don't think Swell is the issues. There are some 3rd party cookies being sent with some swell requests, but they appear to be from other apps. I will reopen this ticket if there is an issue after I do more research.

**Reply by ericingram** [reply]
Good to know, thank you
